.github/workflows/TagBot.yml

```diff
@@ -1,11 +1,15 @@
 name: TagBot
 on:
-  schedule:
-    - cron: 0 * * * *
+  issue_comment:
+    types:
+      - created
+  workflow_dispatch:
 jobs:
   TagBot:
+    if: github.event_name == 'workflow_dispatch' || github.actor == 'JuliaTagBot'
     runs-on: ubuntu-latest
     steps:
       - uses: JuliaRegistries/TagBot@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          ssh: ${{ secrets.DOCUMENTER_KEY }}
```

Comment:
Is there a link for setting this up, this repo doesn't have it.

Reply:
Ugh, I had missed this message before.

You can [create the SSH key](https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#travis-ssh) with `DocumenterTools.jl`:
```julia
using DocumenterTools
DocumenterTools.genkeys(user="JuliaCloud", repo="AWSS3.jl")
```
and add it as a secret in the repository settings (the on-screen instructions will guide you).

Note this is needed only if you want tagging a new release to trigger CI on it (especially useful if you built documentation on CI, which at the moment isn't the case), otherwise you can just ignore it.